sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusReceivedMessage.cs

```diff
@@ -167,12 +167,7 @@ internal ServiceBusReceivedMessage(): this(body: default)
         ///   Optionally describes the payload of the message, with a descriptor following the format of
         ///   RFC2045, Section 5, for example "application/json".
         /// </remarks>
-        public string ContentType
-        {
-            get => AmqpMessage.Properties.ContentType;
```

Comment:
This was a leftover remnant from backing out the MessageWithMetadata experimental changes.